### PR TITLE
Split up ZTF submissions when windows are not contiguous

### PR DESCRIPTION
**Pull request by mcoughlin**
**Does this pull request make any changes to the database?**
No.

Code changes that affect the database require special attention for data migration.
If your change affects the database, describe the migration plan here.

### CODE REVIEW

File: growth/too/views.py [hunk 1]
```diff
@@ -1233,52 +1233,103 @@ def get_json_data(plan, decam_style=True):
...
-                    'exposure_time': exposure.exposure_time/ditherNorm,
-                    'program_pi': program_pis[telescope],
-                    'subprogram_name': "ToO_"+transient_name
-                }
-                for ii, exposure in enumerate(exposures)
-                if exposure.filter_id in
-                exposure.field.reference_filter_ids
-            ]
-        }
+    if telescope == "ZTF":
```
Comment (lpsinger) [inline growth/too/views.py]: Why is this specific to ZTF?

Comment (mcoughlin) [reply growth/too/views.py]: ZTF is the only one that doesnt receive the time of observations in the files to use, so it is what it is ;)

Comment (lpsinger) [reply growth/too/views.py]: Hmm.  Is it possible to at least factor out the common code to reduce duplication?

I get really nervous about having very different code paths for triggering different facilities.

Comment (mcoughlin) [reply growth/too/views.py]: Hmmm the splitting is pretty particular... maybe @shreyasahasram08 has ideas?

**Comment by lpsinger** [comment]
Why was this closed?

**Comment by mcoughlin** [comment]
I was removing old branches

**Pull request closed without merging**